gflownet/gflownet.py

```diff
@@ -424,6 +438,7 @@ def sample_batch(
                         is_forward=True,
                         temperature=1.0,
                         random_action_prob=self.random_action_prob,
+                        mask_invalid_actions=masks_invalid_actions_forward,
```

Comment:
I would simply pass `None` and remove the definition of `masks_invalid_actions_forward = None` in line 428 above.